src/main/java/it/gov/pagopa/payhub/auth/controller/AuthzControllerImpl.java

```diff
@@ -34,6 +41,19 @@ public ResponseEntity<OperatorsPage> getOrganizationOperators(String organizatio
                 .build());
     }
 
+    @Override
+    public ResponseEntity<OperatorDTO> createOrganizationOperator(String organizationIpaCode,
+        CreateOperatorRequest createOperatorRequest) {
+        if(organizationAccessMode){
+            throw new InvalidOrganizationAccessDataException("No organizationAccess information");
+        }
```

Comment:
when organizationAccessMode is enabled, this API should not be invoke.
The error should be something like "API not implemented"
```suggestion
        if(organizationAccessMode){
            new ResponseEntity<>(HttpStatus.NOT_IMPLEMENTED)
        }
```